06-setup-backup-restore/trilio.md

```diff
@@ -261,25 +262,35 @@ If the output looks like above, you installed `TVK` successfully. Next, you will
 By default, when installing `TVK` via `Helm`, there is no `Free Trial` license installed automatically. You can always go to the `Trilio` website and generate a new [license](https://www.trilio.io/plans) for your cluster that suits your needs (for example, you can pick the `basic license` type that lets you run `TrilioVault` indefinetly if your `cluster capacity` doesn't exceed `10 nodes`). A free trial license lets you run `TVK` for `one month` on `unlimited` cluster nodes.
 
 **Notes:**
-- **TrilioVault is free of charge for Kubernetes clusters with up to 10000 nodes for DigitalOcean users.**
+- **TrilioVault is free of charge for Kubernetes clusters with up to 100000 nodes for DigitalOcean users. User can follow below steps to create a license specifically created for DO users**
 - `Starter Kit` examples rely on a `Cluster` license type to function properly.
 
-### Checking TVK Application Licensing
+### Creating and Checking TVK Application Licensing
 
-Please run below command to see if license is available for your cluster (it is managed via the `License` CRD):
+Please run below command to see if license:
```

Comment:
The line updated to as below:
```
Please run below command to create a new license for your cluster (it is managed via the `License` CRD):

```shell
kubectl apply -f 06-setup-backup-restore/assets/manifests/triliovault/tvk_install_license.yaml
```
Since the command actually creates a license and apply it on the TVK instance for the cluster.
Let me know if it reads okay.